sdk/include/psdk/WbemTime.h

```diff
@@ -0,0 +1,135 @@
+/*
+* PROJECT:     ReactOS Kernel
+* LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+* PURPOSE:     Header of WBEMTime and WBEMTimeSpan classes.
+* COPYRIGHT:   Copyright 2017 Jin Juanshi (2033537949@qq.com)
+*/
+
+#pragma once
+
+#ifndef WBEMTIME_HEADERFILE_IS_INCLUDED
+#define WBEMTIME_HEADERFILE_IS_INCLUDED
+
+#include <windows.h>
+#include <provexce.h>
+
+#define INVALID_TIME 0xffffffffffffffff
+
+///////////////////////////////////////////////////////////////////////////
+// WBEMTimeSpan - This class holds time span values. 
+
+class __declspec(dllexport) WBEMTimeSpan
+{
+private:
+
+    ULONGLONG m_Time;
+    friend class WBEMTime;
+
+public:
+
+    WBEMTimeSpan() { m_Time = INVALID_TIME; }
+    WBEMTimeSpan(const BSTR bstrDMTFFormat) { *this = bstrDMTFFormat; }
+
+    WBEMTimeSpan(
+
+        int iDays,
+        int iHours,
+        int iMinutes,
+        int iSeconds,
+        int iMSec = 0,
+        int iUSec = 0,
+        int iNSec = 0
+    );
+
+    WBEMTimeSpan operator+ (const WBEMTimeSpan &uAdd) const;
+    const WBEMTimeSpan &operator+= (const WBEMTimeSpan &uAdd);
+
+    WBEMTimeSpan operator- (const WBEMTimeSpan &uSub) const;
+    const WBEMTimeSpan &operator-= (const WBEMTimeSpan &uSub);
+
+    const WBEMTimeSpan &operator= (const BSTR pDMTFFormat);
+
+    BOOL operator== (const WBEMTimeSpan &a) const { return m_Time == a.m_Time; }
+    BOOL operator!= (const WBEMTimeSpan &a) const { return m_Time != a.m_Time; }
+    BOOL operator<  (const WBEMTimeSpan &a) const { return m_Time < a.m_Time; }
+    BOOL operator<= (const WBEMTimeSpan &a) const { return m_Time <= a.m_Time; }
+    BOOL operator>  (const WBEMTimeSpan &a) const { return m_Time > a.m_Time; }
+    BOOL operator>= (const WBEMTimeSpan &a) const { return m_Time >= a.m_Time; }
```

Comment:
You seem to use a mixture of BOOL and bool. Is it because it's done so in the MS PSDK?

Reply:
I tried to fix entry points of framedyn.dll file. If I change it, entry alias will be corrupted.